test/cmd/admin.sh

```diff
@@ -172,6 +172,7 @@ os::cmd::expect_success 'oc adm policy add-role-to-user view baz --rolebinding-n
 os::cmd::expect_success_and_text 'oc get rolebinding/view -o jsonpath="{.metadata.name},{.roleRef.name},{.userNames[*]}"' '^view,view,foo$'
 os::cmd::expect_success_and_text 'oc get rolebinding/custom -o jsonpath="{.metadata.name},{.roleRef.name},{.userNames[*]}"' '^custom,view,bar baz$'
 os::cmd::expect_failure_and_text 'oc adm policy add-role-to-user other fuz --rolebinding-name=custom' '^error: rolebinding custom found for role view, not other$'
+os::cmd::expect_success_and_text 'oc adm policy remove-cluster-role-from-group self-provisioner system:authenticated:oauth' "Warning: Your changes may get lost whenever a master is restarted, unless you prevent reconciliation of this rolebinding using the following command: oc annotate clusterrolebinding self-provisioners 'rbac.authorization.kubernetes.io/autoupdate=false' --overwrite"
```

Comment:
Needs the `.rbac` postfix or the command will not work (because of the proxy conversion):

```
$ oc get clusterrolebinding.rbac self-provisioners -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    rbac.authorization.kubernetes.io/autoupdate: "true"
  creationTimestamp: 2018-08-29T20:42:25Z
  name: self-provisioners
  resourceVersion: "204"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/self-provisioners
  uid: 095d2136-abcc-11e8-8285-507b9dac97ff
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: self-provisioner
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: system:authenticated:oauth


$ oc annotate clusterrolebinding self-provisioners 'rbac.authorization.kubernetes.io/autoupdate=false' --overwrite
clusterrolebinding.authorization.openshift.io/self-provisioners annotated


$ oc get clusterrolebinding.rbac self-provisioners -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    rbac.authorization.kubernetes.io/autoupdate: "true"
  creationTimestamp: 2018-08-29T20:42:25Z
  name: self-provisioners
  resourceVersion: "204"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/self-provisioners
  uid: 095d2136-abcc-11e8-8285-507b9dac97ff
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: self-provisioner
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: system:authenticated:oauth


$ oc annotate clusterrolebinding.rbac self-provisioners 'rbac.authorization.kubernetes.io/autoupdate=false' --overwrite
clusterrolebinding.rbac.authorization.k8s.io/self-provisioners annotated


$ oc get clusterrolebinding.rbac self-provisioners -o yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  annotations:
    rbac.authorization.kubernetes.io/autoupdate: "false"
  creationTimestamp: 2018-08-29T20:42:25Z
  name: self-provisioners
  resourceVersion: "927"
  selfLink: /apis/rbac.authorization.k8s.io/v1/clusterrolebindings/self-provisioners
  uid: 095d2136-abcc-11e8-8285-507b9dac97ff
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: self-provisioner
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: Group
  name: system:authenticated:oauth
```

Reply:
Maybe we need to make the conversion code better?

Reply:
We should actually run the annotate command right after this test, and then run the test again and make sure it no longer warns.

Reply:
grrr and I had checked it ... oh well

Reply:
Yeah good idea, I'll add the annotate add back the self-provisioner, then take it out again